src/GitLabApiClient/Models/Trees/Requests/TreeQueryOptions.cs

```diff
@@ -0,0 +1,20 @@
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace GitLabApiClient.Models.Trees.Requests
+{
+    public sealed class TreeQueryOptions
+    {
+        public string Branch { get; set; }
```

Comment:
Branch is not accurate, it can be a reference which points to Branch, Tag or commit sha

```suggestion
        public string Reference { get; set; }
```